Gems/Multiplayer/Code/Source/MultiplayerStatSystemComponent.cpp

```diff
@@ -221,9 +222,21 @@ namespace Multiplayer
                     AZStd::for_each(
```

Comment:
Why not just `for (auto& stat : group.m_stats.m_items)`. Ever since ranged for came into the standard, I'm a bit confused why people still use for_each.